packages/actor-http-fetch/test/ActorHttpFetch-test.ts

```diff
@@ -443,5 +443,67 @@ This error can be disabled by modifying the 'httpBodyTimeout' and/or 'httpTimeou
       );
       jest.useRealTimers();
     });
+
+    it('should retry with a delay', async() => {
+      let numberOfRetries = 0;
+      const customFetch = jest.fn(async(_, init) => {
+        if (numberOfRetries < 2) {
+          numberOfRetries++;
+          throw new Error('Retry count not reached.');
+        }
+        return {};
+      });
```

Comment:
Rather than having this logic you could probably just test how many times the fetch function was called https://stackoverflow.com/questions/52885551/jest-check-how-many-times-a-mocked-module-function-was-called

Reply:
Ah I see you have it below - why is this needed as well?

Reply:
Not sure if I'm getting you right here. The point of counting the retries was that I want the fetch function to throw twice but resolve on the third time. The page you linked just talks about testing the number of function calls.

Reply:
Yep - I was just rushed when I did the review and missed the intention. This looks good!